README.md

```diff
@@ -1,36 +1,67 @@
-# leveller.js
+# Leveller
 
-Levels elements per row so that their heights match. Right now it's a jQuery plugin, but a vanilla JS version is planned.
+A jQuery plugin for equalizing element heights.
 
-This is super early right now. You're welcome to give it a whirl, but we don't have any fancy demos or even cross-browser testing yet. Consider this a version `0.1`.
+If you can, [you should use Flexbox instead](http://css-tricks.com/snippets/css/a-guide-to-flexbox/). See the [demo]() for an example.
 
-If you can, [you should use Flexbox instead](http://css-tricks.com/snippets/css/a-guide-to-flexbox/).
+For simple, single-row adjustments, [Equalizer](https://github.com/skrajewski/Equalizer) is a leaner, dependency-free solution.
 
-If you only have a single row or you need to equalize widths, you should try [equalize.js](https://github.com/tsvensen/equalize.js).
+- [Blog post](http://blog.cloudfour.com/leveller/)
+- [Live demo](http://cloudfour.github.io/leveller/)
+
+## Installation
+
+Include the plugin after you've included [jQuery](http://jquery.com/):
+
+```html
+<script src="path/to/jquery.min.js"></script>
+<script src="path/to/leveller.min.js"></script>
+```
+
+Or use [npm](https://www.npmjs.com/) and [Browserify](http://browserify.org/):
+
+```
+npm install --save jquery
+npm install --save leveller
+```
+```javascript
+var $ = require('jquery');
+require('leveller');
+```
 
 ## Usage
 
-Equalize tile heights across rows with default options:
+Equalize element heights across rows:
 
 ```javascript
-$(elements).leveller();
+$('.example .column').leveller();
 ```
 
-Initialize with advanced options:
+Apply adjustments to a property of a child element:
```

Comment:
As someone just jumping in here cold turkey, my first furrowed-brow moment happens here as I'm like "Apply adjustments for...what...exactly?" Is this about setting a given CSS property (here `padding-top`) to be equal across a set of elements in a row? Just wondering if this explanation could use some extra clarity. If it would be obvious to someone wanting to use this plugin, though...rock on.